addon/data/shoot-panel.js

```diff
@@ -346,7 +346,7 @@ class ShootPanel extends React.Component {
 
   addClip() {
     isAdding[this.props.shot.id] = this.props.shot.clipNames().length;
-    setTimeout(renderData);
+    setTimeout(() => renderData);
```

Comment:
Isn't this going to return the renderData function, doing nothing? I think you need to call renderData()